### DISCUSSION TITLE
Allow User to specify Blur/LQIP URL for Next/Image

### DISCUSSION THREAD

**Discussion by dukesx** [vercel/next.js#36001] [Ideas]
### Describe the feature you'd like to request

Change quality of Custom Loader to show LQIP instead of pathetic Data64 that has to be custom generated again and again for every image. 

### Describe the solution you'd like

We live in 2022. Majority uses Cloudinary or Imagekit type of CDN, on the go services for fast image transfer. Wouldn't it be nice if Next Image component could show LQIP using
-  Quality attribute of custom loader? 

Okay, I get it, you are busy but atleast can't you LET US provide the url for LQIP? I hate the fact that I have to generate Data64 url for every image if I want a SIMPLE LQIP. Come on!! In todays date YOU don't need data64 or blurHash, all you need is Cloudinary or Imagekit. Period. 

### Describe alternatives you've considered

React progressive Graceful image and cloudinary advanced image component

**Comment by dukesx** [comment]
Any Input on this ?

**Comment by balazsorban44** [comment]
Thank you for the feature request, _but I would like to remind you of our [Code of Conduct](https://github.com/vercel/next.js/blob/canary/CODE_OF_CONDUCT.md). Let's keep our tones respectful towards each other. There is no need for negative language._ :green_heart:

Are you saying the Cloudinary loader and `blurDataUrl` do not cover your use case?:
https://nextjs.org/docs/api-reference/next/image#built-in-loaders
https://nextjs.org/docs/api-reference/next/image#blurdataurl

If I understand correctly, your suggestion is to make an extra request for the placeholder image, while Next.js currently tries to embed the placeholder at build-time (as base64) in the initial response to make it load faster and avoid a potentially slow origin that would prevent the placeholder to ever show up.

**Reply by dukesx** [reply]
Yes that is exactly what i mean but it has to be understood that Next js generates a base64 IF i.e i import a local image, but what about foreign images ? Yes for that you have to save your base64 in a database and then fetch at run time and then set it as parameter for image.  

This still makes it slow , but with an inconvenience of having to first generate, save and then fetch the base64 for EVERY image i have. I was hoping that you would allow the developer to decide on which strategy for LQIP would be best. Fetch a dynamically generated blur image directly from Cloudinary/ImageKit or generate/save/fetch the base64 ourselves. Its just an open choice that some of us developers require.

**Comment by dmoKruso** [comment]
Totally agree with this. LQIP for placeholder image is almost a standard nowadays would be great to have it included in next image.

**Comment by harrisrobin** [comment]
Would like to vote my support for this as well.
